src/InfoBoxes/Content/Factory.cpp

```diff
@@ -152,7 +152,7 @@ static constexpr MetaData meta_data[] = {
   {
     N_("Thermal climb, last 30 s"),
     N_("TC 30s"),
-    N_("A 30 second rolling average climb rate based of the reported GPS altitude, or vario if available."),
+    N_("A 30 second rolling average climb rate based of the reported GPS altitude, or vario if available. The number in smaller font reflects the climb rate for the current thermal since circling started."),
```

Comment:
Same "de-bloat" here as well.
The phrase "since circling started" is redundant because "current thermal" implies that.